manifests/base/rbac/role.yaml

```diff
@@ -59,21 +59,22 @@ rules:
 - apiGroups:
   - trainer.kubeflow.org
   resources:
-  - trainjobs
+  - trainingruntimes/finalizers
+  - trainjobs/finalizers
+  - trainjobs/status
   verbs:
-  - create
-  - delete
   - get
-  - list
   - patch
   - update
-  - watch
 - apiGroups:
   - trainer.kubeflow.org
   resources:
-  - trainjobs/finalizers
-  - trainjobs/status
+  - trainjobs
   verbs:
+  - create
+  - delete
```

Comment:
Do we need to grant create and delete permission for TrainJob controller since it is user responsibility to create and delete TrainJobs right now ?

Reply:
This is caused by master branch https://github.com/kubeflow/trainer/blob/08232dac6645b7c737cab7c4db6739b7f70efcb4/pkg/controller/trainjob_controller.go#L69

Do you want to open PR? If not, I can open it afterwards.

Reply:
I will fix it.